var/spack/repos/builtin/packages/r-pcapp/package.py

```diff
@@ -38,5 +38,5 @@ class RPcapp(RPackage):
         version('1.9-60', '23dd468abb9fedc11e40166446df1017')
         version('1.9-50', 'be44f173404fd6e86ba0a5515711bfa3')
 
-        depends_on('r@3.4.0:3.4.9')
+        depends_on('r@3.4.0:')
```

Comment:
According to https://cran.r-project.org/web/packages/pcaPP/index.html, there aren't any restrictions on the R version, so you can remove this.